Lombiq.HelpfulExtensions/Extensions/ShapeTracing/ShapeTracingShapeEvents.cs

```diff
@@ -22,6 +22,15 @@ public Task DisplayedAsync(ShapeDisplayContext context)
         var builder = new HtmlContentBuilder(6);
         var shapeMetadata = context.Shape.Metadata;
 
+        var isPageTitle = shapeMetadata.Type == "PageTitle";
+
+        // This is needed to have the shape info as a comment, otherwise it would be put in the title.
+        if (isPageTitle)
+        {
+            builder.AppendHtml(context.ChildContent);
+            builder.AppendHtmlLine("</title>");
```

Comment:
This leaves the original closing tag dangling, which is invalid:
![image](https://github.com/Lombiq/Helpful-Extensions/assets/4007293/0ef8f7ec-08ed-4563-8805-f51f4f6b34b9)

Also it's totally valid to use the `PageTitle` shape in the main document or anywhere else outside of the header's `<title>` element. In that case this closing element will fail the HTML validation.